src/Microsoft.DotNet.Build.Tasks/PackageFiles/tests.targets

```diff
@@ -32,9 +33,11 @@
          that check the invoked process name against known casing which fail under code coverage.
          This is probably because the code coverage target invocation does not canonicalize the
          process name. -->
-    <TestHost Condition="'$(OS)' == 'Windows_NT'">CoreRun.exe</TestHost>
-    <TestHost Condition="'$(OS)' != 'Windows_NT'">corerun</TestHost>
-    <TestHost>$(TestPath)dnxcore50/$(TestHost)</TestHost>
+    <TargetOS Condition="'$(TargetOS)'==''">$(OS)</TargetOS>
```

Comment:
At some point we should probably always default this to "corerun" and only add the "./" in the target where TargetOS will already be defined.